src/index.js

```diff
@@ -187,7 +187,8 @@ export default async function fetch(url, options_) {
 
 			let body = pump(response_, new PassThrough(), reject);
 			// see https://github.com/nodejs/node/pull/29376
-			if (process.version < 'v12.10') {
+			/* c8 ignore next 3 */
+			if (/^v12\.\d\./.test(process.version) || process.version < 'v12') { // i.e. semver <12.10
```

Comment:
Sorry, I didn't get this change. It was comparing with 12.10 earlier and now it's just 12?

Reply:
@tinovyatkin See #1038 for the problem. The solution, here, is the regex, which matches for a **single-digit** minor version of 12, i.e. 12.0.x - 12.9.x. I know it's obtuse, but it seemed more reasonable than adding the  whole semver package just for this one condition.

Reply:
Hmm... Can we construct a RegEx then to cover these conditions without the need for a second check? I mean, it looks redundant to first check the line with RegEx and then with string comparison.... Something like `/^v1([01]|2\.\d)\.d/` should work in a single pass, right (our minimum supported version is 10.x already)?

Reply:
@tinovyatkin Sure, I can do that. I thought was I had done here was simpler, but it seems I was wrong 🙂. I'll make this change. M

Reply:
This change is made.